### DISCUSSION TITLE
koishi-plugin-status 要改名吗？

### DISCUSSION THREAD

**Discussion by shigma** [koishijs/koishi#154] [Show and tell]
下一个大版本的 koishi-plugin-status 将不止包含 status 这个单一的指令，还会包含一个 Web UI。

![image](https://user-images.githubusercontent.com/33423008/111323754-21385d80-86a5-11eb-963b-d723b7214210.png)

↑ 一个非常简陋的 demo

### 仪表盘

一目了然的数据可视化页面，status 指令的加强版。

### 机器人

监控每个机器人的状态，并提供在运行时添加新机器人和停用机器人的可能性。

### 插件

查看当前注册的所有插件，并提供在运行时停用插件的可能性。

### 沙盒

提供一个虚拟的机器人环境，你可以在这个环境中进行调试，而不必去污染生产环境的聊天记录。

### 聊天（TODO）

直接使用机器人的账号今天聊天（注意这个功能其实与上一条是相反的）。

## 是否改名

由于这些功能其实已经超过了 status 的范畴，可能改个名字会更好。为此收集一下大家的想法。目前的提案有：

- status
- webui
- console
- control

### 起名要求

请注意下面的名字不会被接受：

1. 使用其他与本插件无关产品的专有名称
2. 使用自造词
3. 过长的名称

**Comment by idlist** [comment]
而现在， 你改完了
